.copier-answers.yml

```diff
@@ -1,10 +1,11 @@
 # Changes here will be overwritten by Copier
-_commit: ae42d0d
+_commit: v0.3.0
 _src_path: ../copier-core-python-repository
 email: zebedee.nicholls@climate-resource.com
-initial_setup: true
```

Comment:
I think this is what is causing your having to manually copy over the pyproject.toml changes. Why change this? I always use intial_setup: true and it manages to merge in pyproject changes

Reply:
I had some idea that this would cause headaches (e.g. it always creates the `operations.py` file which is annoying if you forget, although not that annoying and generally you notice in review etc.). I've moved this comment to the discussion in the copier repo, it might actually be the answer.